### PR TITLE
Check hwrng test

### PR DESCRIPTION
**Pull request by rickwu666666**
Modify the job of checking rng_current of HWRNG to fit different HWRNG type.
side load result
https://pastebin.canonical.com/p/c5CHgdbvvd/
https://pastebin.canonical.com/p/nPtsFwwTcS/

### CODE REVIEW

**Comment by rickwu666666** [comment]
@stanley31huang @patliuu Thanks for the reviewed. And I have updated the fix.

File: checkbox-provider-ce-oem/units/crypto/accelerator.pxu [hunk 1]
```diff
@@ -48,6 +62,8 @@ command:
         echo "FAIL: CAAM job ring interrupts didn't increase!"
         exit 1
     fi
+requires: manifest.has_caam == 'True'
+imports: from com.canonical.plainbox import manifest
```
Comment (patliuu) [inline checkbox-provider-ce-oem/units/crypto/accelerator.pxu:66]: Could you help to put this 2 lines to the upper of this job? To make the `command:` field be the bottom of the test job. :bow:

Comment (rickwu666666) [reply checkbox-provider-ce-oem/units/crypto/accelerator.pxu]: done. Thanks

**Review by patliuu — APPROVED** [review]
LGTM

File: checkbox-provider-ce-oem/units/crypto/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                        08 ff a6 52 1a 8d 59 fc 83 f3 fb 68 54 3c 56 d3 
+            Digest:     da db 41 09 dc ac e0 3f 9d 56 3b 2e ac 2e 5b 26 
+                        54 06 cd ba 58 52 d2 77 e2 31 c3 60 8f 9b 8b b5 
+```
+### ce-oem-crypto/af_alg{cipher}
+The AF_ALG related jobs aim to test the kernel crypto API with specific ciphers.
+
+## Accelerator Tests
+
+### ce-oem-crypto/caam/rng-available
```
Comment (baconYao) [inline checkbox-provider-ce-oem/units/crypto/README.md:65]: I think this `ce-oem-crypto/caam/rng-available` can be removed. Am I right?

**Review by baconYao — CHANGES_REQUESTED** [review]

File: checkbox-provider-ce-oem/units/crypto/README.md (hunk 2 above)
Comment (rickwu666666) [reply checkbox-provider-ce-oem/units/crypto/README.md]: @baconYao Yes, you are right. I've updated it. Thanks.

Comment (baconYao) [reply checkbox-provider-ce-oem/units/crypto/README.md]: Great.

**Review by baconYao — APPROVED** [review]
LGTM

**Pull request merged**